lib/DependencyScan/ScanDependencies.cpp

```diff
@@ -1100,7 +1100,7 @@ forEachBatchEntry(CompilerInstance &invocationInstance,
       pCache = std::get<2>((*subInstanceMap)[entry.arguments]).get();
       SmallVector<const char *, 4> args;
       llvm::cl::TokenizeGNUCommandLine(entry.arguments, saver, args);
-      CompilerInvocation subInvok = invok;
+      CompilerInvocation subInvok = invoke;
```

Comment:
sometimes groups try to avoid using keywords (like `class`).

I apparently missed `Invok` (I handle cases separately).